### PR TITLE
Experiment in the interactive shell

### PR DESCRIPTION
**Pull request by mhanberg**

### CODE REVIEW

File: _posts/2019-02-18-experiment-in-the-interactive-shell.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+### Avoid Analysis Paralysis
+
+>Analysis paralysis is when the fear of potential error outweighs the realistic expectation or potential value of success, and this imbalance results in suppressed decision-making in an unconscious effort to preserve existing options. [*](https://en.wikipedia.org/wiki/Analysis_paralysis#Software_development)
+
+Idle hands build nothing. I fell into this trap the other day, wasting hours before I remembered the power of the interactive shell.
+
+I knew I needed to use either a [Supervisor](https://hexdocs.pm/elixir/Supervisor.html) or a [Dynamic Supervisor](https://hexdocs.pm/elixir/DynamicSupervisor.html), but I wasn't sure which one was right for the job.
+
+Unclear on how to proceed, **I spent a few hours** reading documentation, searching the internet for blog posts, and asking people for advice. Did I learn enough to make a decision? _No_.
```
Comment (swanson) [inline _posts/2019-02-18-experiment-in-the-interactive-shell.md]: "searching for blog posts"

File: _posts/2019-02-18-experiment-in-the-interactive-shell.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+} 
+```
+
+I find experimenting in the shell to be useful when I need a **short feedback loop** that isn't tightly coupled to the full stack of my application.
+
+### Avoid Analysis Paralysis
+
+>Analysis paralysis is when the fear of potential error outweighs the realistic expectation or potential value of success, and this imbalance results in suppressed decision-making in an unconscious effort to preserve existing options. [*](https://en.wikipedia.org/wiki/Analysis_paralysis#Software_development)
+
+Idle hands build nothing. I fell into this trap the other day, wasting hours before I remembered the power of the interactive shell.
```
Comment (swanson) [inline _posts/2019-02-18-experiment-in-the-interactive-shell.md]: "I fell into this trap recently,"

**Comment by mhanberg** [comment]
@swanson any thoughts on a better name? I am not a fan of the word `experiment` because I think I use it too much.

Also `interactive shell` seems like a mouthful, but just `shell` doesn't describe the right thing.

**Comment by swanson** [comment]
REPL?

**Comment by swanson** [comment]
experiment -> test out
experiment -> try out

Dunno if it is much better

**Pull request merged**